app/services/import_reach_skills_volunteer_opportunities.rb

```diff
@@ -60,4 +60,14 @@ def populate_vol_op_attributes(op, model, coordinates)
     model.updated_at = one_day_ago # https://www.pivotaltracker.com/story/show/153805125
     model.created_at = one_day_ago
   end
+
+  private
+    def assign_coordinate lat_lng, coords
+      return coords.send(lat_lng).to_f if coords
+      if lat_lng == :latitude
+        Location::CENTRAL_HARROW_LATITUDE
+      else
+        Location::CENTRAL_HARROW_LONGITUDE
+      end
+    end
```

Comment:
@tansaku I'm not really sure if this is better, but code climate likes it. What do you think?